### PR TITLE
DigitalOcean Wiki Updates - CMake, G++7, and other Edits

### PR DESCRIPTION
**Pull request by heyaapl**
- Updated the initial application install command to add g++-7
- Updated the initial application install command to remove CMake and Make
- Added a section explaining how to install CMake and Make
- Clarified the directions of some parts of the tutorial

### CODE REVIEW

**Comment by heyaapl** [comment]
I've verified every step in this tutorial today to ensure it works, and was able to make a few additional edits to the wiki not related to required G++7 and CMake version edits, such as adjusting how the Data files are downloaded and clarifications in other steps. This should be good to merge - the video is slightly out of date, but should serve as a sufficient guide as I preface the written tutorial should be followed directly.

File: docs/digital-ocean-video-tutorial.md [hunk 1]
```diff
@@ -108,21 +132,20 @@ GRANT ALL ON *.* TO 'dbadmin'@'%' IDENTIFIED BY 'password1' WITH GRANT OPTION;
...
 ```
 MTHREADS=`grep -c ^processor /proc/cpuinfo`; MTHREADS=$(($MTHREADS + 2));
 make -j $MTHREADS;
 make install -j $MTHREADS;
 ```
+- After the terminal says `make install -j $MTHREADS` after compiling, hit 'Return' or 'Enter' on your Keyboard to install and finish the compiling process.
 
 #### Data Files
-- Let's navigate back to the home directory `cd`
+- As the AZCore User, let's navigate back to the home directory `cd`
```
Comment (BarbzYHOOL) [inline docs/digital-ocean-video-tutorial.md]:
```suggestion
- As the `azcore` user, let's navigate back to the home directory `cd`
```

**Comment by BarbzYHOOL** [comment]
good work

**Comment by FrancescoBorzi** [comment]
good job @heyaapl

**Pull request merged**